src/primitive_crypto.cc

```diff
@@ -24,6 +24,45 @@
 
 namespace toit {
 
+enum class AesTypes {
+  Ecb,
+  Cbc
+};
+
+static Object* aes_crypt(Process* process, AesEcbContext* context, Blob* input, int from, int to, bool encrypt, AesTypes type) {
+  if (from < 0 || to > input->length() || from > to || ((to - from) % 16) != 0) INVALID_ARGUMENT;
+
+  Error* error = null;
+  ByteArray* result = process->allocate_byte_array(to - from, &error);
+  if (result == null) return error;
+
+  ByteArray::Bytes output_bytes(result);
+
+  switch (type)
+  {
+  case AesTypes::Ecb:
+    mbedtls_aes_crypt_ecb(
+      &context->context_,
```

Comment:
4 space indent of arguments.

Reply:
What do you mean here? That there should be 4 spaces of indentation or that there is 4 spaces?
I have made the indentations 2 spaces for now.